saskatoon/sitebase/templates/app/detail_views/harvest/status.html

```diff
@@ -47,14 +82,46 @@ <h2><span class="counter">{{ requests.count }}</span>
                 <div class="wb-traffic-inner notika-shadow sm-res-mg-t-30 tb-res-mg-t-30 dk-res-mg-t-30">
                     <div class="website-traffic-ctn">
                         <p>{% trans "Pick leader" %}</p>
-                        {% if harvest.pick_leader != None %}
-                        <h2>{{ harvest.pick_leader }}</h2>
+                        {% if harvest.pick_leader is None %}
+                        <div class="row">
+                            <div class="col-lg-8">
+                                <h2>{% trans "Missing" %}</h2>
+                            </div>
+                            <div class="col-lg-4 breadcomb-report">
+                                <a href="{% url 'harvest-adopt' harvest.id %}">
+                                    <button
+                                    data-placement="left"
+                                    title="Adopt it!"
+                                    class="btn"
+                                    >
+                                        <strong>{% trans "Adopt it!" %}</strong>
+                                    </button>
+                                </a>
+                            </div>
+                        </div>
+                        {% elif harvest.pick_leader.id == request.user.id %}
+                        <div class="row">
+                            <div class="col-lg-7">
+                                <h2>{{ harvest.pick_leader.name }}</h2>
+                            </div>
+                            <div class="col-lg-5 breadcomb-report">
+                                <a href="{% url 'harvest-leave' harvest.id %}">
+                                    <button
+                                    data-placement="left"
+                                    title="Leave it!"
+                                    class="btn"
+                                    >
+                                        <strong>{% trans "Leave it!" %}</strong>
```

Comment:
I would personally use the same formatting as the Edit harvest button (currently not bold)